### ISSUE TITLE
Ability to add labels/comments to linked issues/PRs

### ISSUE THREAD

**Issue by agkphysics** [python-semantic-release/python-semantic-release#975]
## Description

The NodeJS implementation of semantic-release has the ability to automatically tag relevant PRs and issues with specified labels with the [`@semantic-release/github`](https://github.com/semantic-release/github) plugin. It would be useful for this Python version to also be able to do so.

## Use cases

You can then easily see which issues and PRs have been released, and on which versions. Technically this is already possible for PRs by looking at the tags associated with the merge commit, but that requires extra steps.

## Possible implementation

Since there is planned functionality for linking PRs into changelogs, presumably said PRs could be passed to a step that adds labels to them, or adds a comment on them.

Issues may be slightly more difficult as they are linked via PRs.

**Comment by codejedi365** [comment]
@agkphysics, I agree it is a very nice feature that I have been doing manually for this project because of how nice it is. It is an eventual goal but we have some minor stuff to solve first

**Comment by codejedi365** [comment]
still valuable, no effort towards implementation has been made yet.

**Comment by codejedi365** [comment]
Setting some ground work for this feature by enabling commit parsers to extract the issue numbers and PR numbers from commit messages.

**Comment by codejedi365** [comment]
Under initial development and design, will require a lot more API definitions for a set of issue tracking systems. Significant amount of work to be done.

**Comment by codejedi365** [comment]
Still in backlog

**Comment by codejedi365** [comment]
no movement here yet, still in backlog.

**Comment by codejedi365** [comment]
Still in backlog

**Comment by lowlydba** [comment]
@codejedi365 I see there's a somewhat-mature, somewhat-related open PR for this still - any updates or clarity you can provide? I've found this to be a high value quality of life feature for many engineers
